### PR TITLE
Remove monorepo

### PR DESCRIPTION
**Pull request by jssblck**
# Overview

Removes monorepo from FOSSA CLI, except in a couple places where we currently test for whether a project is monorepo before attempting to wait for its build or uploading a new scan.

Also removed:

- Documentation about the `fossa vps` subcommand.
- The `wiggins` embedded binary.
- API calls, modules, and functions that were only related to monorepo/VPS.

## Acceptance criteria

Monorepo no longer exists in FOSSA CLI, and meanwhile everything else still works.
I'm relying on tests and the type system to prove the latter.

## Testing plan

I built the project and was able to run a simple scan, otherwise relying on tests.

## Risks

No users should be depending on this functionality any longer, so this should not be risky.

## References

https://fossa.atlassian.net/browse/ANE-755

## Checklist

- [x] I added tests for this PR's change (or explained in the PR description why tests don't make sense).
- [x] If this PR introduced a user-visible change, I added documentation into `docs/`.
- [x] If this change is externally visible, I updated `Changelog.md`. If this PR did not mark a release, I added my changes into an `# Unreleased` section at the top.
- [x] If I made changes to `.fossa.yml` or `fossa-deps.{json.yml}`, I updated `docs/references/files/*.schema.json`. You may also need to update these if you have added/removed new dependency type (e.g. `pip`) or analysis target type (e.g. `poetry`).

### CODE REVIEW

**Comment by jssblck** [comment]
@zlav planning to resolve the conflict later, whenever I merge both my outstanding PRs.

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # FOSSA CLI Changelog
 
+## v3.7.12
```
Comment (zlav) [inline Changelog.md:3]: Should we pair this PR with first-party license scanning and make the release 3.8? The deprecation of monorepo (breaking change) and a significant new feature seem like a good reason to bump the minor version.

@spatten

**Review by zlav — APPROVED** [review]
Well that was an easy review! This cleans up a LOT

File: Changelog.md (hunk 1 above)
Comment (spatten) [reply Changelog.md:3]: That works for me, as long as we're fine with waiting until the first-party license scanning is done

Comment (jssblck) [reply Changelog.md:3]: @spatten please just ping me (or feel free to just resolve conflicts & merge this PR) whenever that is done then!

Comment (jssblck) [reply Changelog.md]: As discussed in slack, merging after all to make #1203 simpler.

**Pull request merged**